rasa_core/domain.py

```diff
@@ -196,9 +229,13 @@ def collect_intent_properties(intent_list):
         intent_properties = {}
         for intent in intent_list:
             if isinstance(intent, dict):
+                for properties in intent.values():
+                    if 'use_entities' not in properties:
+                        properties['use_entities'] = True
                 intent_properties.update(intent)
             else:
-                intent_properties.update({intent: {'use_entities': True}})
+                intent = {intent: {'use_entities': True}}
+                intent_properties.update(intent)
```

Comment:
any reason this isn't just what it was originally? `intent_properties.update({intent: {'use_entities': True}})`

Reply:
Yes, I'm not 100% sure if it's necessary but the reason was that this function modifies the intents. Previously those intents that did not have properties added in the domain would not have properties after this function was called either, even though they are supposed to have the property `use_entities`.
It might not matter though because properties of intents are afaik always accessed through the `domain.intent_properties` dictionary.